Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/AppContentView.swift

```diff
@@ -51,8 +53,7 @@ struct AppContentView: View {
                     Label("My Apps", systemImage: "network")
                 }
 
-            if Purchases.isConfigured {
-
+            if false && Purchases.isConfigured {
```

Comment:
same thought here

Reply:
in terms of what to do with these... I'd just kill them. The App tab was a workaround because we didn't have log in / log out, and the Upsell tab was just a quick way to show a single paywall, but it's fairly useless

Reply:
(I'd probably do it on a separate PR though)

Reply:
OK, comment left, will delete in future!